.travis.yml

```diff
@@ -81,6 +81,7 @@ jobs:
         - export IPFS_JS_EXEC=`pwd`/src/cli/bin.js
         - export IPFS_REUSEPORT=false
         - cd node_modules/ipfs-interop
+        - npm install go-ipfs-dep
```

Comment:
Since this is in the `devDependencies` map, these lines can be removed, right?

Reply:
they can now yes